src/form-field/interfaces.ts

```diff
@@ -67,6 +72,13 @@ export interface FormFieldProps extends BaseComponentProps {
   errorText?: React.ReactNode;
 }
 
+export interface I18nStrings {
```

Comment:
I think this should be moved into a separate `FormFieldProps` namespace to make sure this is 100% compatible with our documenter and website. See for example wizard as a reference: https://github.com/cloudscape-design/components/blob/main/src/wizard/interfaces.ts#L103

Reply:
Just pushed an update with this @timogasda